Fix docker-entrypoint.sh not found in $PATH

FedericoAntoniazzi opened the pull request:
`docker-entrypoint.sh` wasn't executable so the container fails initialization giving this error:

_ERROR: for docker-spigot_1  Cannot start service spigot: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"docker-entrypoint.sh\": executable file not found in $PATH": unknown_

FedericoAntoniazzi committed:
Fix docker-entrypoint.sh not found in $PATH

docker-entrypoint wasn't executable so the container fails initialization giving this error:

ERROR: for prova_spigot_1  Cannot start service spigot: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"docker-entrypoint.sh\": executable file not found in $PATH": unknown

FedericoAntoniazzi committed:
Fix entrypoint path

D3strukt0r commented:
Looks good to me, I'll do the merge tomorrow if you don't have anything to add any more. Shouldn't it also work just with `ENTRYPOINT ["docker-entrypoint.sh"]`? That's how many other public repositories have it (e. g. [wordpress](https://github.com/docker-library/wordpress/blob/master/php7.4/fpm-alpine/Dockerfile)). I might consider trying it out without the path (just so it looks nicer)

FedericoAntoniazzi commented:
In the ENTRYFILE you have to specify shell scripts or commands you want to execute.
I think that just the name it's nicer too but it's indifferent. 

I have an idea to make it look even more nicer so wait a commit before merging

D3strukt0r commented:
I've added a fix for Travis, or at least I think it should fix it. So that your builds pass. Maybe you have to merge my develop to your fork. No idea actually, first time doing this :P

FedericoAntoniazzi committed:
Fix permission problems

Changelog:
- Fix entrypoint filepath
- Fix End Of Lines: Unix EOL is different from Windwos' one

Problem fixed:
standard_init_linux.go:211: exec user process caused "no such file or directory"

Note: To replace the EOL I've used dos2unix
